frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -722,6 +722,20 @@ describe("scenarios > question > filter", () => {
     });
   });
 
+  it("should offer case expression in the auto-complete suggestions", () => {
+    openReviewsTable({ mode: "notebook" });
+    cy.findByText("Filter").click();
+    cy.findByText("Custom Expression").click();
+    popover().contains(/case/i);
+
+    // "case" is still there after typing a bit
+    cy.get("[contenteditable='true']")
+      .as("inputField")
```

Comment:
No need to alias it if we're not going to reuse it in this test.

Reply:
Good point! I've removed the alias.